quesma/queryparser/pancake_transformer.go

```diff
@@ -70,6 +74,24 @@ func (a *pancakeTransformer) metricAggregationTreeNodeToModel(previousAggrNames
 	}, nil
 }
 
+func (a *pancakeTransformer) pipelineAggregationToLayer(previousAggrNames []string, pipeline *pancakeAggregationTreeNode) (layer *pancakeModelPipelineAggregation, err error) {
+	if pipeline == nil {
+		return nil, fmt.Errorf("pipeline aggregation is nil")
+	}
+
+	pipelineQueryType, ok := pipeline.queryType.(model.PipelineQueryType)
+	if !ok {
+		return nil, fmt.Errorf("pipeline aggregation %s is not pipeline aggregation, type: %s", pipeline.name, pipeline.queryType.AggregationType().String())
+	}
+
+	return &pancakeModelPipelineAggregation{
+		name:         pipeline.name,
+		internalName: fmt.Sprintf("pipeline__%s", strings.Join(append(previousAggrNames, pipeline.name), "__")),
```

Comment:
It could be empty as we don't have column.

Reply:
Hmm, I don't think I understand. `pipeline.name` is always non-empty, isn't that so?